ignition/resource_ignition_config.go

```diff
@@ -11,12 +11,12 @@ import (
 
 var configReferenceResource = &schema.Resource{
 	Schema: map[string]*schema.Schema{
-		"source": &schema.Schema{
```

Comment:
@appilon I'm wondering about this change. Is it really needed?
I know Go 1.11+ supports this syntax, which is really nice. I do feel though that in Terraform land, where everything is being cast back and forth to `interface{}` it's easy to lose track of the actual types of things (at least mentally) and switching to the new literal syntax doesn't help with that. 
May I suggest we stick to the old literal syntax that highlights the types better? Unless of course, there are plans to deprecate it in the language.